### PR TITLE
Update DB maintenance window

### PR DESCRIPTION
**Pull request by sbuffo**

### CODE REVIEW

**Comment by sbuffo** [comment]
retest this please

File: release/cloudformation/av-sched-stores.template [hunk 1]
```diff
@@ -110,12 +110,12 @@
...
                 "BackupRetentionPeriod"      : 7,
                 "Engine"                     : "mysql",
-                "EngineVersion"              : "5.6.19a",
+                "EngineVersion"              : "5.6.34",
                 "LicenseModel"               : "general-public-license",
                 "MasterUsername"             : { "Ref": "DBUser" },
                 "MasterUserPassword"         : { "Ref": "DBPwd" },
                 "PreferredBackupWindow"      : "04:28-04:58",
-                "PreferredMaintenanceWindow" : "sun:01:55-sun:02:25",
+                "PreferredMaintenanceWindow" : "mon:14:15-mon:14:45",
```
Comment (nbosansky) [inline release/cloudformation/av-sched-stores.template]: is it in UTC or local time of the region?

Comment (sbuffo) [inline release/cloudformation/av-sched-stores.template]: indeed, ...

Comment (sbuffo) [inline release/cloudformation/av-sched-stores.template]: "The weekly time range (in UTC) during which system maintenance can occur.", so we have a problem :-(

Comment (nbosansky) [inline release/cloudformation/av-sched-stores.template]: we can use a cloudformation parameter to set the maintenance windows and choose the value accordingly to the region

Comment (sbuffo) [inline release/cloudformation/av-sched-stores.template]: yes, that's what I was thinking, I will do it

**Comment by sbuffo** [comment]
retest this please

**Pull request merged**